draft-ietf-httpbis-digest-headers.md

```diff
@@ -1059,11 +1064,32 @@ in conjuction with the encrypted content-coding {{?RFC8188}}.
 
 ## Algorithm Agility
 
-...
+The security properties of digest-algorithms are not fixed.
+Algorithm agility is achieved establishing the IANA Digest Algorithm Values
+registry (see {{iana-digest-algorithm-registry}}) and Section 2.1 of {{?RFC7696}}).
+
+To help endpoints make secure algorithm choices,
```

Comment:
Let's not say secure, because algorithms will inevitably be broken. How about 

```suggestion
To help endpoints understand weaker algorithms from stonger ones,
```